tests/test_config_parsing/test_parser_error_collection.py

```diff
@@ -840,7 +838,7 @@ def test_that_non_readable_workflow_job_is_localized():
             line=3,
             column=19,
             end_column=42,
-            filename="exists_but_not_runnable",
+            filename="test.ert",
```

Comment:
The line and column information refers to this file, so it needs to be the filename for linting to work with IDEs.